tests/test_main.py

```diff
@@ -0,0 +1,12 @@
+from cookiecutter.main import is_repo_url
+
+
+def test_is_repo_url():
+    """Verify is_repo_url works."""
+    assert is_repo_url('gitolite@server:team/repo') is True
+    assert is_repo_url('git@github.com:audreyr/cookiecutter.git') is True
+    assert is_repo_url('https://github.com/audreyr/cookiecutter.git') is True
+    assert is_repo_url('gh:audreyr/cookiecutter-pypackage') is True
```

Comment:
I understand the thoroughness concern, but isn't that overkill since there's no semantic difference with the test testing `https://github...`?